### PR TITLE
Travis CI Testing

### PR DESCRIPTION
**Pull request by dgasmith**
Adds Travis CI testing for psi4numpy. Currently it simply runs the scripts and ensures that they exit with a 0 flag. More comprehensive testing can be added in the future.

- [ ] Does anyone know how to get Travis to print colors? This seems to work fine on my screen, but not there.

### CODE REVIEW

File: .scripts/travis_test.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    nleft = nequals // 2
+
+    print(("=" * (nleft + nequals %2)) + title + ("=" * nleft))
+
+def print_flag(success):
+    if success:
+        print("\033[92m PASSED\033[00m")
+    else:
+        print("\033[91m FAILED\033[00m")
+
```
Comment (loriab) [inline .scripts/travis_test.py]: maybe it needs the escape? Psi's color shows up in travis `${Esc}[1;34m`

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - hash -r
+  - conda config --set always_yes yes --set changeps1 no
+  - conda update -q conda
+  - conda info -a
+  - conda create -q -n p4env python=3.5 psi4 scipy runipy -c psi4
+  - source activate p4env
+  - eval "$(~/Gits/psixc/objdir/stage/usr/local/psi4/bin/psi4 --psiapi-path)"
```
Comment (loriab) [inline .travis.yml]: This path is on travis?

File: Moller-Plesset/MP3_SO.py [hunk 3]
```diff
@@ -82,6 +83,9 @@
 print('MP2 correlation energy:      %16.10f' % MP2corr_E)
 print('MP2 total energy:            %16.10f' % MP2total_E)
 
+# Compare to Psi4
+psi4.driver.p4util.compare_values(psi4.energy('MP2'), MP2total_E, 6, 'MP2 Energy')
+
```
Comment (loriab) [inline Moller-Plesset/MP3_SO.py]: maybe just `psi4.compare_values()`?

File: .scripts/travis_test.py (hunk 1 above)
Comment (loriab) [inline .scripts/travis_test.py]: Esc is itself a variable. Here's the file (https://github.com/psi4/psi4/blob/master/cmake/custom_color_messages.cmake)

File: .travis.yml (hunk 2 above)
Comment (dgasmith) [inline .travis.yml]: Ohyea, not needed.

File: .scripts/travis_test.py (hunk 1 above)
Comment (dgasmith) [inline .scripts/travis_test.py]: What color is that?

File: Moller-Plesset/MP3_SO.py (hunk 3 above)
Comment (dgasmith) [inline Moller-Plesset/MP3_SO.py]: Fixed in all cases.

**Comment by dgasmith** [comment]
@loriab @dsirianni Can we get this reviewed and in so that others are not continuously failing.

File: Self-Consistent-Field/SAD/input.dat [hunk 4]
```diff
@@ -0,0 +1,149 @@
+# A simple Psi 4 input script to compute a SCF reference
+# Requires umpy 1.7.2+
```
Comment (loriab) [inline Self-Consistent-Field/SAD/input.dat]: typo

Comment (dgasmith) [inline Self-Consistent-Field/SAD/input.dat]: Thats not supposed to be there yet ;0

**Pull request merged**